crates/broker/src/order_picker.rs

```diff
@@ -30,6 +30,14 @@ use boundless_market::{
 use thiserror::Error;
 use tokio::task::JoinSet;
 
+use OrderPricingOutcome::{Lock, ProveImmediate, Skip};
+
+// fraction the stake the protocol gives to the prover who fills an order that was locked by another prover but expired
+// e.g. a value of 1 means 1/4 of the original stake is given to the prover who fills the order.
+// This is determined by the constant SLASHING_BURN_BPS defined in the BoundlessMarket contract.
+// The value is 4 because the slashing burn is 75% of the stake, and we give the remaining 1/4 of that to the prover.
+const FRACTION_STAKE_REWARD: u64 = 4;
```

Comment:
This is fine for now, but we should really fetch this from the contract. I created a task here: https://github.com/boundless-xyz/boundless/issues/517 Could you add a TODO?

Reply:
done